library/c/sandesh.h

```diff
@@ -25,6 +25,52 @@ extern "C" {
 #include <Wsk.h>
 #include <Ws2ipdef.h>
 
+static void *
+s_malloc(unsigned int size)
+{
+    void *mem = ExAllocatePoolWithTag(NonPagedPool, size+1, 'RVCO');
```

Comment:
You can move `'RVCO'` tag into a global variable as in `windows/vrouter_mod.c` in vRouter code.

```
ULONG  SxExtAllocationTag = 'RVCO';
```